openapi.yaml

```diff
@@ -1,46 +1,92 @@
 swagger: '2.0'
 info:
-  title: Express API Skeleton
-  description: Skeleton for Express APIs
+  title: Corvallis Parks API
+  description: an api for the parks of Corvallis, OR
   version: v1
   license:
     name: GNU Affero General Public License Version 3
     url: http://www.gnu.org/licenses/agpl-3.0.en.html
   contact:
-    name: IS Data Architecture Team
-    url: https://is.oregonstate.edu/data-architecture
-    email: isdataarchitecture@oregonstate.edu
+    name: David Kalcic
+    email: kalcicd@oregonstate.edu
 schemes:
   - https
 host: api.oregonstate.edu
 basePath: /v1
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/express-api-skeleton
+  url: https://github.com/osu-mist/parks-api
 produces:
   - application/json
 paths:
-  /pets:
+  /parks:
     get:
-      summary: List all pets
+      summary: List all parks
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+      - parks
+      description: Get all parks
+      operationId: getParks
       parameters:
         - $ref: '#/parameters/authorization'
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
         - in: query
-          name: species
-          description: Search by pet species
+          name: Amenities
+          description: Filter by park amenities ("play area", "hiking trails", etc.)
+          required: false
+          type: array
```

Comment:
For this array query parameter, could you specify `collectionFormat`? We usually use `csv` for this